internal/otelcollector/config/metric/agent/relabel_configs.go

```diff
@@ -130,17 +130,17 @@ func dropIfIstioProxy() RelabelConfig {
 	}
 }
 
-func dropIfSchemeAnnotationHTTP(annotated AnnotatedResource) RelabelConfig {
+func dropIfSchemeAnnotationHTTP() RelabelConfig {
```

Comment:
```suggestion
func dropIfSchemeHTTP() RelabelConfig {
```

There is no tight coupling to the annotation here

Reply:
Yeah, the name also has to be changed